services/summarizer/standard/parameters.go

```diff
@@ -104,6 +108,25 @@ func WithValidatorSummaries(enabled bool) Parameter {
 	})
 }
 
+// WithvalidatorRetainPubkeys states if the module should retain balance and epoch summaries for a subset of validator.
+func WithvalidatorRetainPubkeys(validatorRetainPubkeys []string) Parameter {
```

Comment:
This should have a capital "v".

Also, the pubkeys should not be passed in as a string slice.  This function should accept `[]BLSPubKey` to allow for catching and reporting errors in the input.

Reply:
Updated !